src/main/java/io/iron/ironmq/Queue.java

```diff
@@ -785,59 +779,23 @@ public QueueModel update(QueueModel model) throws IOException {
     }
 
     /**
-     * Add alerts to a queue. If there is no queue, an EmptyQueueException is thrown.
-     * @param alerts The array list of alerts.
      * @throws io.iron.ironmq.HTTPException If the IronMQ service returns a status other than 200 OK.
      * @throws java.io.IOException If there is an error accessing the IronMQ server.
      */
```

Comment:
lines that should be deleted as well

Reply:
Can't find these. Did you delete it already?